<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>587</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sistema de Automação e Controle Residencial</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="parar_musica_pushButton">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    font: 11pt &quot;Tibetan Machine Uni&quot;;
    border-radius: 10px;       /* Bordas arredondadas */
    background-color: rgb(255, 255, 255); /* Fundo semi-transparente */
	border: 1px solid rgb(0,0,0);
}

/* Efeito de escurecimento ao passar o mouse */
QPushButton:hover {
    background-color: rgba(225, 225, 225, 0.7); /* Cor mais escura ao passar o mouse */
}
</string>
      </property>
      <property name="text">
       <string>⏸️ Música</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="locale">
       <locale language="Portuguese" country="Brazil"/>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1" rowspan="2">
         <widget class="Line" name="vertical1_line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="controlar_eletronicos_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    font: 11pt &quot;Tibetan Machine Uni&quot;;
    border-radius: 10px;       /* Bordas arredondadas */
    padding: 5px;              /* Espaço interno */
    background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */
	border: 1px solid rgb(0,0,0);
}

/* Efeito de escurecimento ao passar o mouse */
QPushButton:hover {
    background-color: rgba(225, 225, 225, 0.7); /* Cor mais escura ao passar o mouse */
}
</string>
          </property>
          <property name="text">
           <string>Controle de Eletrônicos</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="visualizar_dados_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Tibetan Machine Uni&quot;;
border-radius: 10px;       /* Bordas arredondadas */
padding: 5px;              /* Espaço interno */
background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */</string>
          </property>
          <property name="text">
           <string>Exibe Dados de Todos os Sensores:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="visualizar_dados_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    font: 11pt &quot;Tibetan Machine Uni&quot;;
    border-radius: 10px;       /* Bordas arredondadas */
    padding: 5px;              /* Espaço interno */
    background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */
	border: 1px solid rgb(0,0,0);
}

/* Efeito de escurecimento ao passar o mouse */
QPushButton:hover {
    background-color: rgba(225, 225, 225, 0.7); /* Cor mais escura ao passar o mouse */
}
</string>
          </property>
          <property name="text">
           <string>Visualização de dados em tempo real</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="controlar_eletronicos_label">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Tibetan Machine Uni&quot;;
border-radius: 10px;       /* Bordas arredondadas */
padding: 5px;              /* Espaço interno */
background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */</string>
          </property>
          <property name="text">
           <string>Permite o Controle de Eletrônicos:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="bem_vindo_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>900</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>900</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tibetan Machine Uni</family>
            <pointsize>30</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;Tibetan Machine Uni&quot;;
border-radius: 10px;       /* Bordas arredondadas */
padding: 5px;              /* Espaço interno */
background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Seja bem-vindo(a) ao nosso sistema!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="Line" name="horizontal1_line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pag_anterior2_pushButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    font: 11pt &quot;Tibetan Machine Uni&quot;;
    border-radius: 10px;       /* Bordas arredondadas */
    padding: 5px;              /* Espaço interno */
    background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */
	border: 1px solid rgb(0,0,0);
}

/* Efeito de escurecimento ao passar o mouse */
QPushButton:hover {
    background-color: rgba(225, 225, 225, 0.7); /* Cor mais escura ao passar o mouse */
}
</string>
          </property>
          <property name="text">
           <string>Voltar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="info_pag2_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1500</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tibetan Machine Uni</family>
            <pointsize>30</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;Tibetan Machine Uni&quot;;
border-radius: 10px;       /* Bordas arredondadas */
padding: 5px;              /* Espaço interno */
background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */

</string>
          </property>
          <property name="text">
           <string>Aqui você pode visualizar os dados coletados pelos sensores:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="Line" name="line_4">
          <property name="minimumSize">
           <size>
            <width>1000</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;Tibetan Machine Uni&quot;;
border-radius: 10px;       /* Bordas arredondadas */
padding: 5px;              /* Espaço interno */
background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */

</string>
          </property>
          <property name="text">
           <string>Aqui você pode controlar os equipamentos:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pag_anterior3_pushButton">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    font: 11pt &quot;Tibetan Machine Uni&quot;;
    border-radius: 10px;       /* Bordas arredondadas */
    padding: 5px;              /* Espaço interno */
    background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */
	border: 1px solid rgb(0,0,0);
}

/* Efeito de escurecimento ao passar o mouse */
QPushButton:hover {
    background-color: rgba(225, 225, 225, 0.7); /* Cor mais escura ao passar o mouse */
}
</string>
          </property>
          <property name="text">
           <string>Voltar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;Tibetan Machine Uni&quot;;
border-radius: 10px;       /* Bordas arredondadas */
padding: 5px;              /* Espaço interno */
background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */</string>
            </property>
            <property name="text">
             <string>Fan</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="ligarLED_pushButton">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    font: 11pt &quot;Tibetan Machine Uni&quot;;
    border-radius: 10px;       /* Bordas arredondadas */
    padding: 5px;              /* Espaço interno */
    background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */
	border: 1px solid rgb(0,0,0);
}

/* Efeito de escurecimento ao passar o mouse */
QPushButton:hover {
    background-color: rgba(225, 225, 225, 0.7); /* Cor mais escura ao passar o mouse */
}
</string>
            </property>
            <property name="text">
             <string>Ligar</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="ligarFAN_pushButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    font: 11pt &quot;Tibetan Machine Uni&quot;;
    border-radius: 10px;       /* Bordas arredondadas */
    padding: 5px;              /* Espaço interno */
    background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */
	border: 1px solid rgb(0,0,0);
}

/* Efeito de escurecimento ao passar o mouse */
QPushButton:hover {
    background-color: rgba(225, 225, 225, 0.7); /* Cor mais escura ao passar o mouse */
}
</string>
            </property>
            <property name="text">
             <string>Ligar</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;Tibetan Machine Uni&quot;;
border-radius: 10px;       /* Bordas arredondadas */
padding: 5px;              /* Espaço interno */
background-color: rgba(255, 255, 255, 0.6); /* Fundo semi-transparente */</string>
            </property>
            <property name="text">
             <string>Led</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tibetan Machine Uni</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu_Op_es">
    <property name="title">
     <string>&amp;Opções</string>
    </property>
    <addaction name="actionSalvar_gr_ficos"/>
   </widget>
   <addaction name="menu_Op_es"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>242</red>
         <green>229</green>
         <blue>142</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>242</red>
         <green>229</green>
         <blue>142</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>145</red>
         <green>145</green>
         <blue>145</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Tibetan Machine Uni</family>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Sobre"/>
   <addaction name="separator"/>
   <addaction name="action_Reposit_rio_Git_do_Projeto"/>
   <addaction name="separator"/>
   <addaction name="actionSalvar_gr_ficos"/>
  </widget>
  <action name="action_Sobre">
   <property name="icon">
    <iconset theme="dialog-information">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Sobre</string>
   </property>
  </action>
  <action name="action_Reposit_rio_Git_do_Projeto">
   <property name="icon">
    <iconset theme="emblem-symbolic-link">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Repositório Git do Projeto</string>
   </property>
  </action>
  <action name="actionSalvar_gr_ficos">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Salvar Gráficos</string>
   </property>
   <property name="toolTip">
    <string>Salvar gráficos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
